mail/api.py

```diff
@@ -173,13 +215,14 @@ def send_financial_aid_email(cls, acting_user, financial_aid, subject, body):
             financial_aid (FinancialAid): the FinancialAid object this pertains to (recipient is pulled from here)
             subject (str): email subject
             body (str): email body
+            raise_for_status (bool): If true and we received a non 2xx status code from Mailgun, raise an exception
         Returns:
             requests.Response: response from Mailgun
         """
         from_address = cls.default_params()['from']
         to_address = financial_aid.user.email
-        response = cls.send_individual_email(subject, body, to_address)
-        if response.status_code == HTTP_200_OK:
+        response = cls.send_individual_email(subject, body, to_address, raise_for_status=raise_for_status)
+        if response.status_code == status.HTTP_200_OK:
```

Comment:
Why even check the status code here? `raise_for_status()` will ensure that either the response was successful, or an exception will be raised so that this code won't even execute. (Is the caller prepared for the possibility that this function will raise an exception?)